grype/presenter/cyclonedx/vulnerability.go

```diff
@@ -52,60 +54,73 @@ type Advisories struct {
 	Advisory []string `xml:"v:advisory"`
 }
 
+// cvssVersionToMethod accepts a CVSS version as string (e.g. "3.1") and converts it to a
+// CycloneDex rating Method, for example "CVSSv3"
```

Comment:
nit: CycloneDex -> CycloneDx

Reply:
adjusted in https://github.com/anchore/grype/pull/317/commits/528d45eaa973efc85357734adbfa5588dac6dde8